app.py

```diff
@@ -1,9 +1,9 @@
 from flask import Flask, request, jsonify
 from blueprints.ridhwan import ridhwan
-
+from blueprints.ranim import ranim
+from blueprints.ranim2 import ranim2
 app = Flask("Testing app")
-app.register_blueprint(ridhwan)
-
```

Comment:
You should not remove my blueprint. Anyway, all changes to app.py should not be pushed to master so I will remove them. No action required.